src/main/java/liquibase/ext/spanner/change/AddLookupTableChangeSpanner.java

```diff
@@ -36,7 +37,7 @@
 
   @Override
   public boolean supports(Database database) {
-    return (database instanceof CloudSpanner);
+    return (database instanceof ICloudSpanner);
```

Comment:
Checking against `ICloudSpanner` (the interface) instead of `CloudSpanner` ensures that changes that generate a proxy for the database are still considered Cloud Spanner databases.